[Angular] rootNamespace' of type 'string | undefined' is not assignable to string index type 'string'.

* **3.1.0-rc3**.
* Angular (Abp Ng 3.1.0-rc3)
* error TS2411: Property 'rootNamespace' of type 'string | undefined' is not assignable to string index type 'string'.
* Build Angular app with @abp/ng.core

```
  > ng build "--prod"

  Compiling angular-oauth2-oidc : es2015 as esm2015
  Compiling @abp/utils : es2015 as esm2015
  Compiling @abp/ng.core : es2015 as esm2015

  WARNING in Exceeded maximum budget for RELATIVE_FOLDER_HERE/AngularApp/src/app/app.component.scss. Budget 6 kB was not met by 1.76 kB 
with a total of 7.76 kB.

  ERROR in node_modules/@abp/ng.core/lib/models/config.d.ts:27:9 - error TS2411: Property 'rootNamespace' of type 'string | undefined' is not assignable to string index type 'string'.

  27         rootNamespace?: string;
             ~~~~~~~~~~~~~

  npm ERR! code ELIFECYCLE
  npm ERR! errno 1
  npm ERR! APPNAME_HERE@0.0.1 build: `ng build "--prod"`
  npm ERR! Exit status 1
  npm ERR!
  npm ERR! Failed at the APPNAME_HERE@0.0.1 build script.
  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

  npm ERR! A complete log of this run can be found in:
  npm ERR!     CACHE_PATH_HERE\npm-cache\_logs\2020-08-31T11_15_50_324Z-debug.log
PROJECT_NAME_HERE.csproj(52,5): error MSB3073: The command "npm run build -- --prod" exited with code
```

--- a/npm/ng-packs/packages/core/src/lib/models/config.ts
+++ b/npm/ng-packs/packages/core/src/lib/models/config.ts
@@ -22,11 +22,12 @@ export namespace Config {
     logoUrl?: string;
   }
 
-  export interface ApiConfig {
+  export type ApiConfig = {
     [key: string]: string;
-    rootNamespace?: string;
     url: string;
-  }
+  } & Partial<{
+    rootNamespace: string;
+  }>;
 
   export interface Apis {
     [key: string]: ApiConfig;

